app/jobs/risc_delivery_job.rb

```diff
@@ -23,7 +23,7 @@ def perform(
     event_type:,
     issuer:,
     now: Time.zone.now
-  )
+  )    
```

Comment:
ditto... trailing spaces
```suggestion
  )
```